deploy.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+aws s3 sync . s3://BUCKET/PATH --exclude ".*"
```

Comment:
Depending how you are planning to manage the bucket properties...
But perhaps could be a good change